From a5a459a5bd0f2c00e46147bccead22c1544decea Mon Sep 17 00:00:00 2001
From: Andrew Comminos <andrew@comminos.com>
Date: Sun, 1 Apr 2018 02:16:50 -0500
Subject: [PATCH 5/9] wined3d: Disable persistently mapped shader resource
 buffers.

---
 dlls/wined3d/buffer.c | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/dlls/wined3d/buffer.c b/dlls/wined3d/buffer.c
index aee4b5b356..31dd88e7b4 100644
--- a/dlls/wined3d/buffer.c
+++ b/dlls/wined3d/buffer.c
@@ -1596,6 +1596,10 @@ static HRESULT buffer_init(struct wined3d_buffer *buffer, struct wined3d_device
         {
             WARN_(d3d_perf)("Not creating a persistent mapping for a dynamic buffer because ARB_buffer_storage is unsupported.\n");
         }
+        else if (bind_flags & WINED3D_BIND_SHADER_RESOURCE)
+        {
+            FIXME_(d3d_perf)("Not using a persistent mapping for shader resource buffer %p (unimplemented)\n", buffer);
+        }
         else
         {
             // If supported, use persistent mapped buffers instead of a
-- 
2.11.0

